openlibrary/templates/books/edit.html

```diff
@@ -34,13 +36,16 @@
         else:
             title = _("Editing...")
             note = ""
-    <h2 class="editFormTitle">$:title</h2>
     $if ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians')):
         <span class="adminOnly right">
             <form method="post" id="delete-record" name="delete">
                 <button type="submit" name="_delete" value="true" id="delete-btn">$_("Delete Record")</button>
             </form>
         </span>
+        $:macros.databarEdit(work)
```

Comment:
```suggestion
        $:macros.databarEdit(work)
    $if ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians')):
        <span class="adminOnly right">
            <form method="post" id="delete-record" name="delete">
                <button type="submit" name="_delete" value="true" id="delete-btn">$_("Delete Record")</button>
            </form>
        </span>
```
Lets move the databarEdit up so that it is:
1. Looks the same as the author edit page
2. Is shown to people who aren't librarians